Library/Formula/osquery.rb

```diff
@@ -44,6 +44,7 @@ def install
                                  "--record=installed.txt"}
     end
 
+    mkdir_p "#{buildpath}/generated"
```

Comment:
We would have to cut a new release to add it and we're not ready to do that. For now, I've added an issue to track this and we'll be sure to update for 1.4.2: https://github.com/facebook/osquery/issues/773